app.py

```diff
@@ -1,12 +1,41 @@
-from flask import Flask, render_template, request, redirect, url_for
+from flask import Flask, redirect, render_template, request, url_for
 import session_items as session
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
+
 @app.route('/')
-def index():
-    return 'Hello World!'
+def get_items():
+    return render_template('index.html', todos=session.get_items())
+
+
+@app.route('/item/<id>')
+def get_item(id):
+    item = session.get_item(id)
+    return render_template('oneItemDisplay.html', item=item)
+
+@app.route('/', methods=['POST'])
+def add_item():
+    session.add_item(request.form.get("title"))
+    return redirect("/")
+
+
+@app.route('/save', methods=['POST'])
+def save_item(item):
+    item = session.save_item(item)        
```

Comment:
Item will need to read off the form data. However given you already have working functionality for this in the method above I'd just remove the entire method and the corresponding section in index.html